src/analysis/calculus/deriv.lean

```diff
@@ -994,3 +994,19 @@ lemma deriv_within_pow (hxs : unique_diff_within_at 𝕜 s x) :
 by rw [differentiable_at_pow.deriv_within hxs, deriv_pow]
 
 end pow
+
+/-- If the domain has dimension one, then Fréchet derivative is equivalent to the classical
+definition with a limit. -/
```

Comment:
Could you emphasize that in `has_deriv_at_filter_iff_tendsto` division by zero is not a problem, but here it is a problem so you really need to take the limit along points that are different from `x`?